app/categorytree/CategoryTreeProvider.java

```diff
@@ -1,7 +1,7 @@
-package inject;
+package categorytree;
 
 import com.google.inject.Provider;
-import controllers.RefreshableCategoryTree;
+import inject.SunriseInitializationException;
```

Comment:
Now that the initialization of different classes are distributed outside inject, it would certainly make sense.